src/py/flwr_example/pytorch_minimal/cifar.py

```diff
@@ -152,3 +137,16 @@ def test(
             correct += (predicted == labels).sum().item()
     accuracy = correct / total
     return loss, accuracy
+
+def main():
+    DEVICE = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
+    print('Central PyTorch Training')
+    print('Load data')
+    trainloader, testloader = load_data()
+    print('Start training')
+    train(net= Net(), trainloader=trainloader, epochs=2, device= DEVICE)
+    print('Start Testing')
+    print('Loss and accuracy', test(net= Net(), testloader=testloader, device= DEVICE))
```

Comment:
Move the call to `test()` on a separate line so that readers can see what the return values are